### PR TITLE
Fixing PHPUNIT execution - remove dependency to Contacts app

### PR DESCRIPTION
**Pull request by DeepDiver1975**
@georgehrke @raghunayyar please have a look - THX

NOTE:
- [ ] OCP\IAddressBook misses the method lastModified() - I'll take care and make up my mind on how to implement this

### CODE REVIEW

**Comment by georgehrke** [comment]
Thx :blush: 
I'll check asap

File: backend/contact/object.php [hunk 1]
```diff
@@ -203,19 +206,19 @@ private function getObjectFromContact(AddressBookEntity $addressBook, ContactEnt
...
+			'uid' => $contact['ID'],
 			'rrule' => 'FREQ=YEARLY'
 		]);
 
-		$object = $this->factory->createEntity($vobject, ObjectFactory::FORMAT_VObject);
+		$object = $this->factory->createEntity($vObject, ObjectFactory::FORMAT_VObject);
 		$object->setCalendar($this->calendar);
 		$object->setUri($this->getUriFromContact($addressBook, $contact));
-		$object->setEtag($contact->getETag());
+		$object->setEtag($contact['ETAG']);
```
Comment (georgehrke) [inline backend/contact/object.php:225]: is ETAG always set?

**Comment by georgehrke** [comment]
looks good to me otherwise, thx for helping with rework :blush:

File: backend/contact/object.php (hunk 1 above)
Comment (DeepDiver1975) [inline backend/contact/object.php:225]: that's something we need to double check

**Comment by DeepDiver1975** [comment]
@georgehrke let's move this in - better then nothing :speak_no_evil:

**Comment by georgehrke** [comment]
If i remember correctly this broke the backend api because some undefined methods were called, I'll report back asap.

**Pull request merged**